inventory/sample/group_vars/all.yml

```diff
@@ -43,6 +43,9 @@ extra_agent_args: >-
 # image tag for kube-vip
 kube_vip_tag_version: "v0.5.7"
 
+# image tag for kured
+kured_tag_version: "1.12.0"
```

Comment:
Can/should we make the installation of kured optional?

Reply:
> Yes we could, I am thinking adding a when  statement **kured_tag_version** is defined to include the task in the playbook instead managing two separate variables. How would that be for an implementation?